direct/structures/internal/structures.go

```diff
@@ -1044,8 +1044,18 @@ type TPMSSchemeHash struct {
 	HashAlg TPMIAlgHash
 }
 
+// TPM represents a TPMS_SCHEME_ECDAA.
```

Comment:
nit: "TPMSSchemeECDAA represents a ..."

All Go documentation comments are complete sentences beginning with the thing being documented - https://go.dev/blog/godoc

Reply:
@chrisfenner 

Oops...It was my mistake, thanks.
And I fixed it.